transportable-udfs-avro/src/test/java/com/linkedin/transport/avro/TestAvroWrapper.java

```diff
@@ -171,29 +171,32 @@ public void testMapType() {
 
   @Test
   public void testRecordType() {
-    Schema field1 = createSchema("field1", "\"int\"");
-    Schema field2 = createSchema("field2", "\"double\"");
+    Schema field1 = createSchema("fieldOne", "\"int\"");
```

Comment:
just want to confirm that the change in fieldNames was a cosmetic change and that field names can still be alphanumeric.

Reply:
yes, that's correct. The cosmetic change is for making this a more defensive test w.r.t. case sensitivity.